plugins/LilyPad/LilyPad.cpp

```diff
@@ -1342,7 +1350,7 @@ u8 CALLBACK PADpoll(u8 value)
             // QUERY_DS2_ANALOG_MODE
             case 0x41:
                 // Right?  Wrong?  No clue.
-                if (pad->mode == MODE_DIGITAL || pad->mode == MODE_PS1_MOUSE) {
+                if (pad->mode == MODE_PS1_MOUSE || pad->mode == MODE_NEGCON) {
```

Comment:
`pad->mode == MODE_DIGITAL` has been removed. Is this intentional?

Apart from that, the code seems fine.